3p/integration.js

```diff
@@ -576,6 +579,21 @@ function observeIntersection(observerCallback) {
 }
 
 /**
+ * Registers a callback for position of this iframe with the current
+ * viewport.
+ * @param {function(*)} observerCallback
+ * @returns {!function()} A function which removes the event listener that
+ *    observes for intersection messages.
+ */
+function observePosition(observerCallback) {
```

Comment:
@zhouyx Do we need this for first cut? I rather not officially expose PositionObserver to any 3P yet.